Packs/FeedMalwareBazaar/Integrations/MalwareBazaarFeed/MalwareBazaarFeed.py

```diff
@@ -66,7 +68,19 @@ def main() -> None:
     }
     params['data'] = {'query': 'get_recent',
                       'selector': 'time'}  # query params to get only the recent changes for the incremental feed
-    feed_main(params, 'MalwareBazaar Feed', 'malwarebazzar')
+    try:
+        feed_main(params, 'MalwareBazaar Feed', 'malwarebazzar')
+    except Exception as e:
+        # Handles known server side and network exceptions
+        msg = str(e)
+        if demisto.command() == 'fetch-indicators' and (("502 Server Error: Bad Gateway") in msg
+                                                        or "Connection broken: IncompleteRead" in msg):
+            demisto.error(f"An Error Occurred during the run of the 'fetch-indicators' command: {msg}.")
+            demisto.error("Retrying in 10 seconds...")
+            sleep(10)
+            feed_main(params, 'MalwareBazaar Feed', 'malwarebazzar')
```

Comment:
how do we make sure the retry will happen only one time?

Reply:
It won't.